src/components/if-no-vr-headset.js

```diff
@@ -2,7 +2,7 @@
 var utils = AFRAME.utils;
 
 /**
- * Set properties if headset is not connected by checking getVRDisplays().
+ * Set properties if headset is not connected by checking isSessionSupported().
```

Comment:
we can get rid of this component if we can incorporate the logic in `paint-controls` as described above